<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Basic_Calculator">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Basic Calculator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_X">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;X=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Y">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Y=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_Field">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_Field">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="senh_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>senh(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cosh_Btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cosh(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tanh_Btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tanh(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asen_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>asen(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan_Btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>atan(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos_Btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>acos(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sen_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sen(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan_Btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tan(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sec_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sec(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cot_Btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cot(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csc_Btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>csc(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos_Btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cos(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="invx_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yrtx_Btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>y√x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ln(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logy_Btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>logy(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log10_Btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>log10(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrtx_Btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>√x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xpowy_Btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exp_Btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>exp(x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xfact_Btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_Answer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Answer =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ResultOp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Output Field&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="help_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>80</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>210</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Funtras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pi_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven_Btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight_Btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine_Btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six_Btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five_Btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four_Btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three_Btn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two_Btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="one_Btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero_Btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="x_Btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="y_Btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negative_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pi_half_Btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PI/2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
